released/discovery_center/mission_3501/step1/main.tf

```diff
@@ -246,30 +244,32 @@ resource "btp_subaccount_subscription" "event_mesh_application" {
 resource "btp_subaccount_role_collection_assignment" "event_mesh_admin" {
   depends_on           = [btp_subaccount_entitlement.event_mesh_application]
   for_each             = toset(var.event_mesh_admins)
-  subaccount_id        = data.btp_subaccount.project.id
+  subaccount_id        = data.btp_subaccount.dc_mission.id
   role_collection_name = "Enterprise Messaging Administrator"
+  origin               = local.origin_key_app_users
   user_name            = each.value
 }
 
 resource "btp_subaccount_role_collection_assignment" "event_mesh_developer" {
   depends_on           = [btp_subaccount_entitlement.event_mesh_application]
```

Comment:
same here, it should depends on btp_subaccount_subscription instead of btp_subaccount_entitlement (otherwise the Role Collection "Enterprise Messaging Developer" might not be available and will result in API error)